### PR TITLE
[DOC] simplify fragments

### PR DESCRIPTION
**Pull request by eseiler**
Blocked by #2765

### CODE REVIEW

File: doc/fragments/@target_alphabet@_implicit_conversion_from_@source_alphabet@.md.in [hunk 1]
```diff
@@ -0,0 +1,22 @@
+Normally, we do not allow implicit conversion of single argument constructors, but in this case we make an exception,
+because seqan3::${target_alphabet} and seqan3::${source_alphabet} are interchangeable as they behave nearly the same (e.g. same ranks, same
+char to rank conversion).
+<br>
```
Comment (marehr) [inline doc/fragments/@target_alphabet@_implicit_conversion_from_@source_alphabet@.md.in:4]: Can we also do it without `<br>` ?

We have done it before without the `br`, is this needed now? I thought this is normal markdown

Comment (eseiler) [reply doc/fragments/@target_alphabet@_implicit_conversion_from_@source_alphabet@.md.in:4]: Didn't seem like it, if there is just a new line, there are other margins...
I can have another look, and maybe report it.
This PR can also go into master as it is not important right now.

Comment (marehr) [reply doc/fragments/@target_alphabet@_implicit_conversion_from_@source_alphabet@.md.in:4]: Alright :)

**Review by marehr — APPROVED** [review]
Thank you!!

**Review by Irallia — APPROVED** [review]

**Pull request merged**